README.md

```diff
@@ -294,6 +293,7 @@ The following snippets are available to help (press `Ctrl+Space` for IntelliSens
 * `markdownlint-disable-file`
 * `markdownlint-enable-file`
 * `markdownlint-configure-file`
+* `markdownlint-disable-next-line`
```

Comment:
Nitpick: Please move this after `markdownlint-enable` to match the order at https://github.com/DavidAnson/markdownlint#configuration